changelog_unreleased/flow/13687.md

```diff
@@ -0,0 +1,27 @@
+#### Remove support for Flow comments (#13687 by @thorn0)
```

Comment:
@thorn0 This should be marked as `[BREAKING]`?

Reply:
Yes, let's do that.

Reply:
Done in https://github.com/prettier/prettier/pull/13703